tutorials/yolo_e2e.py

```diff
@@ -57,7 +60,19 @@ def run_inference(onnx_model_file: Path):
 if __name__ == '__main__':
     # YOLO version. Tested with 5 and 8.
     version = 8
```

Comment:
Do we need this anymore?

Reply:
The script would allow them to switch back to 5?

Reply:
more that we don't have an arg to set the version so separating it from the default model name doesn't seem to add much value. don't care too much about this though.